Podfile.lock

```diff
@@ -1,27 +0,0 @@
-PODS:
```

Comment:
Don't we want to keep this if the podfile is staying?

Reply:
Never mind... See there's one in the SampleApp dir.

Reply:
There were two, including an old unused version in `SampleApp`. `Podfile` + `.lock` now live in `SampleApp`.